.gitlab-ci.yml

```diff
@@ -163,3 +173,10 @@ Production:HL:
   only:
     - main
   when: manual
+
+Live:wp40:HL:
+  extends: .deploy-wp40
+  environment:
+    name: wp40-hl-prod
+  rules:
+    - if: '$CI_DEPLOY_FREEZE == null && $CI_COMMIT_BRANCH == "k8s-migration"'
```

Comment:
For testing the branch to be checked against CI_COMMIT_BRANCH should be the branch name, in this case it is ```k8s-tools-api```
```- if: '$CI_DEPLOY_FREEZE == null && $CI_COMMIT_BRANCH == "k8s-tools-api"'```
and it should be reverted to master/main before merging